front/lib/auth.ts

```diff
@@ -233,23 +223,13 @@ export class Authenticator {
 
     let groups: GroupResource[] = [];
     let subscription: SubscriptionType | null = null;
-    let flags: WhitelistableFeature[] = [];
 
     if (workspace) {
-      [groups, subscription, flags] = await Promise.all([
+      [groups, subscription] = await Promise.all([
         user?.isDustSuperUser
           ? GroupResource.superAdminFetchWorkspaceGroups(user, workspace.id)
           : [],
         subscriptionForWorkspace(renderLightWorkspaceType({ workspace })),
-        (async () => {
-          return (
-            await FeatureFlag.findAll({
-              where: {
-                workspaceId: workspace?.id,
-              },
-            })
-          ).map((flag) => flag.name);
-        })(),
```

Comment:
we were quite creative to fetch the feature flag :)

version 1: the anonymous async function call